EAR-1414-move-delete-options-not-visible-on-answer-options

ashtuck opened the pull request:
What is the context of this PR?

Fix for the button not appearing until it was hovered over. The problem was that the colour has been changed to white. Have changed it back to what it was, but it looks like there was a design change. Don’t know if this matches what the current design is supposed to me. But it’s a temporary fix until Monday.

How to review

Launch Author and double check that the button are visible and functioning when adding Checkbox Answers.

What to do after everything is green
    *   Bring this branch up-to-date with Origin/Master
    *   If there are a lot of commits or some are not helpful to read, squash them down
    *   Click the Merge button on this pull request
    *   Does this change mean the Capability examples questionnaire should be updated?
    *   Move the Jira ticket for this task into the next stage of the process

MattElcock reviewed (changes requested):
Hey Tucker,

Thanks for looking at this.

Unfortunately, these buttons are also used within an answer editor, where they need to be white. So, changing the colour back to `colors.secondary` isn't the fix we need. Or, it is, but an extra bit of work is needed to make it so we can recolour the buttons where we need to.

Here's an example of where the answer editor buttons are (currently) (the toolbar next to 'Number answer'):

![image](https://user-images.githubusercontent.com/15129656/125249028-ab5ba000-e2ec-11eb-9621-1b7ced5cc25c.png)

Here's a screenshot of what they should be:

![image](https://user-images.githubusercontent.com/15129656/125249342-0b524680-e2ed-11eb-85bc-9533e315801c.png)

ashtuck commented:
Changed the `DeleteButton` & `MoveButton` components to accept a `color` prop. Right now just accepts `white` and `secondary` but defaults to `secondary` which is the most common use through out the app.

Changed the `AnswerEditor` to reflect the white design.
